poetry.lock

```diff
@@ -294,13 +456,13 @@ optional = false
 python-versions = ">=3.6.0"
 
 [package.extras]
-unicode_backport = ["unicodedata2"]
+unicode-backport = ["unicodedata2"]
```

Comment:
Please make sure you're on the latest poetry version. I believe latest uses underscores, not dashes. Let's try and keep noise in the lock as low as possible.

Reply:
Sorted via Discord. Looks like the latest version of Poetry (1.2.2) actually does use dashes. Hopefully this will not be an issue anymore in the future.